.github/ISSUE_TEMPLATE/vep.md

```diff
@@ -56,11 +56,25 @@ Link to any enhancement PRs.
 See https://github.com/kubevirt/community/blob/main/design-proposals/feature-lifecycle.md#releases for more context, please include links to relevant PRs
 -->
 
-* [] Alpha (v1.5.0?)
-* [] Beta (v1.6.0?)
-  * [] Additional Beta criteria
-* [] GA (v1.7.0?)
-  * [] Additional GA criteria
+* Alpha release target (x.y):
+* Beta release target (x.y):
+* Stable release target (x.y):
```

Comment:
This is for a target graduation. Targets are aspirations that might not happen in reality.
Do we want to add another section for the **current** phase?

Reply:
I think the current phase should be in the issue. The VEP is targeted to a specific phase.